pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -626,8 +626,11 @@ func (k *Kubernetes) CreateKubernetesObjects(name string, service kobject.Servic
 
 	// Check to see if Docker Compose v3 Deploy.Mode has been set to "global"
 	if service.DeployMode == "global" {
-		log.Warning("Global mode not yet supported, containers will only be replicated once throughout the cluster. DaemonSet support will be added in the future.")
-		replica = 1
+		opt.CreateD = false
+		opt.CreateDS = true
+		if opt.Controller != "" && opt.Controller != "daemonset" {
+			log.Fatal("Global deploy mode can only convert to daemonset")
```

Comment:
I think fatal log is not a good idea. May be we should try to let user proceed anyway, wether use deployment( ignore deploy mode) or use ds(ignore flags). I think there should be some existing example to consult in the code.

Reply:
Thank you for review code on weekend. Already change code to use `daemonset` controller as default when global mode get in.   @hangyan